src/Dockerfile.dagster-manylinux-builder

```diff
@@ -1,6 +1,6 @@
 # Builds ghcr.io/daster-io/dagster-manylinux-builder:*
 
-# This docker image contains the PEX builder (builder.pex) and is capable of 
+# This docker image contains the PEX builder (builder.pex) and is capable of
```

Comment:
Might as well fix the outdated comment:

This docker image contains the *dagster-cloud package* and is ...

Reply:
We're copying builder.pex below but dont really use it anymore. I think we should remove it, given new versions of dagster-cloud directly reference the a [manylinux tag](https://github.com/dagster-io/dagster-cloud/blob/38b4be7c4073c74de4e45c291907521d2dcd2d20/dagster-cloud-cli/dagster_cloud_cli/core/docker_runner.py#L7) and not latest. So we will not need to move `latest` and (very) old versions of dagster-cloud that use builder.pex will continue to work.